### PR TITLE
Emil_AIsoc

### PR DESCRIPTION
**Pull request by EmilMed**
I have added information about myself to the About Us section in AIsoc website.

### CODE REVIEW

File: src/data/committee.js [hunk 1]
```diff
@@ -101,6 +101,14 @@ const committee = {
           ai_concept: 'Generative AI.',
           fact: 'I like taking photos too!',
         },
+        {
+          name: 'Emil Medzhidov',
+          position: 'Development officer',
+          image: 'emil.medzhidov.jpg',
+          course: 'BEng Chemical Engineering (1st Year)',
+          ai_concept: 'Robotics',
```
Comment (TheRootOf3) [inline src/data/committee.js]: Hey, please add some sort of punctuation at the end, maybe an emoji? 🤖

**Review by TheRootOf3 — CHANGES_REQUESTED** [review]
Hey, please combine both PRs (metadata + image) together in this one! Otherwise, looks good! 🚀

File: src/data/committee.js (hunk 1 above)
Comment (EmilMed) [reply src/data/committee.js]: Sure, the changes have been added!

**Review by TheRootOf3 — APPROVED** [review]
LGTM! 🥳

**Pull request merged**